web/src/app/views/camp-view/camp-view.component.html

```diff
@@ -114,7 +114,13 @@ <h2 class="mb-3">Táborový život</h2>
 <!-- Camp Info -->
 <section class="py-4">
     <div class="container">
-        <h2 class="text-center mb-3">Na kolik mě ta legrace přijde?</h2>
+      <h2 class="text-center mb-3">Podpora našich partnerů</h2>
+      <p>Děkujeme firmě <a href="https://www.forestlaan.cz/" target="_blank">Forestlaan</a> za jejich podporu a za
+        poskytnutí ostrova, na kterém můžeme pořádat naše tábory. Rádi bychom také podpořili nadaci
+        <a href="https://women-for-women.cz/" target="_blank">Women for Women</a>, která se zasazuje o zlepšení života
+        žen a dětí.</p>
```

Comment:
Prkotina: Jaký máme vlastně přesně plán ohledně podpory W4W? Pokud nějaký konkrétní, nedáme tam spíš 
```suggestion
        poskytnutí ostrova, na kterém můžeme pořádat naše tábory. Podporujeme také organizaci
        <a href="https://women-for-women.cz/" target="_blank">Women for Women</a>, která pomáhá ohroženým
        dětem.</p>
```
?

Reply:
> Prkotina: Jaký máme vlastně přesně plán ohledně podpory W4W? Pokud nějaký konkrétní, nedáme tam spíš 
> ```suggestion
>         poskytnutí ostrova, na kterém můžeme pořádat naše tábory. Podporujeme také organizaci
>         <a href="https://women-for-women.cz/" target="_blank">Women for Women</a>, která pomáhá ohroženým
>         dětem.</p>
> ```
> ?

Plán je že pokud rodiče přispějí na W4W 200 Kč a prokážou to výpisem tak dostanou 200 Kč slevu na tábor. Chceme aby W4W vidělo platby od většího množství lidí a ne jednorázově jen od nás, takže se k tomu co jsem popsal výše nějak rozumně dotlačit (navíc dar se dá odečíst z daní 🙃)

Nicméně to bych chtěl vložit až k informacím ohledně ceny tábora na 2025.

Reply:
> Nicméně to bych chtěl vložit až k informacím ohledně ceny tábora na 2025.

👍 Jasně, koukám, že mají eshop, kde to ale začíná na 500,- příspěvku a pak prostě transparentní účty (pro každé projekty zvlášť, protože 100% darů jde na pomoc, režii hradí zakladatelé, což je zase fajn). Přímou platbou na transparentní účet bude asi trochu krkolomná pro ten daňový odpočet, ale třeba to potvrzení vystaví, když je bude člověk uhánět.

Nicméně to je out of scope. :)